src/Microsoft.Diagnostics.TestHelpers/TestConfiguration.cs

```diff
@@ -359,19 +364,74 @@ public class TestConfiguration
     {
         const string DebugTypeKey = "DebugType";
         const string DebuggeeBuildRootKey = "DebuggeeBuildRoot";
-
+        
         internal static readonly string BaseDir = Path.GetFullPath(".");
+        private static readonly Regex versionRegex = new Regex(@"^(\d+\.\d+\.\d+)(-.*)?", RegexOptions.Compiled);
+        
+        private ReadOnlyDictionary<string, string> _settings;
+        private readonly string _configStringView;
+        private readonly string _truncatedRuntimeFrameworkVersion;
 
-        private Dictionary<string, string> _settings;
 
         public TestConfiguration()
```

Comment:
Does anybody actually use this constructor? It doesn't look it is used in the diagnostics repo. What about in the dotnet-diagnostictests?  We may want to delete it. I'm not sure the use of an empty TestConfiguration.

Reply:
I just left it for back compat. There's a lot to clean here, so I can either add that change in this or do it in a separate PR

Reply:
I wouldn't worry about remove it for now.